src/cmdline.rs

```diff
@@ -409,6 +409,9 @@ pub struct CommonCustomizeConfig {
     // Undocumented option; --dest-device fails without it
     #[structopt(long, hidden = true)]
     pub overwrite_dest_device: bool,
+    /// NetworkManager keyfile for live & dest
+    #[structopt(long, number_of_values = 1, value_name = "path")]
+    pub network_keyfile: Vec<String>,
```

Comment:
This makes sense to me. Wanted to mention there's https://github.com/coreos/coreos-installer/issues/545 also related to this. In the future, we could have this knob actually use that so that it takes effect at live Ignition time.

Reply:
@bgilbert Does this imply something like `--network-keyfile=./foo.nmconnection --network-keyfile=./boo.nmconnection` i.e. allowing it to be used more than once?

@jlebon, does #545 want the network to be applied in the initramfs of the live boot?

Reply:
> @jlebon, does #545 want the network to be applied in the initramfs of the live boot?

Yup exactly. I've talked with @bgilbert about this and we'll try to squeeze it in as part of this.

Reply:
@dustymabe Yes, almost all of the options can be repeated.